iati/home/static/home/js/anchor.js

```diff
@@ -0,0 +1,66 @@
+const React = window.React;
```

Comment:
I'm not very familiar with React (and don't think that @allthatilk is either), so I'd suggest for maintainability we add more comments to this code.

Reply:
To be honest, @dalepotter 99% of this is copy-paste from the Wagtail docs example. http://docs.wagtail.io/en/v2.0/advanced_topics/customisation/extending_draftail.html

Reply:
just looking at this one again, I still think it's worth adding a short comment to the top of the file to explain what the function of this file is, plus [the link](http://docs.wagtail.io/en/v2.0/advanced_topics/customisation/extending_draftail.html#creating-new-entities) to the sample script.

Reply:
@dalepotter the link to the top of the file was there already. I suppose it wasn't viewable through this comment window? I've also added a few lines of description to go with it.

Reply:
Sorry I didn't see that first change - somehow I didn't see the initial changes made to the file in the PR diff. Maybe I didn't hard refresh...

Extra comment looks good though 👍